well-formed.rb

```diff
@@ -0,0 +1,30 @@
+require 'pry'
+
+class StringValidator
+
+  def validate(string)
+    trimmed_string = string.gsub(/[^\[\]\(\)\{\}]/,'')
+    if trimmed_string.size.odd?
+      return false
+    elsif trimmed_string.empty?
+      return true
+    else
+      return even_string_validator(string)
+    end
+  end
+  
+  def even_string_validator(string)
+    symbols_to_check = { '{' => '}', 
```

Comment:
Is there a better way to store this to make it more reusable? Its this mutable? Should it be?